src/pages/_app.tsx

```diff
@@ -1,16 +1,17 @@
 import { AppProps } from 'next/app';
 
-import Connect from '@scrtsybil/src/components/connect';
+import Layout from '@scrtsybil/src/components/Layout';
 import { ContextProvider } from '@scrtsybil/src/context';
 
-import '@scrtsybil/src/styles.css';
 import 'antd/dist/antd.dark.min.css';
+import '@scrtsybil/src/styles.css';
 
 const MyApp = ({ Component, pageProps }: AppProps) => {
   return (
     <ContextProvider>
-      {!process.env.IN_PROGRESS && <Connect />}
```

Comment:
by deleting this, we are allowing the prod site to have buttons we don't want. Please use process.env.IN_PROGRESS to hide or show anything on the prod site. (It's set to true for prod only).